ui/dashboards/src/components/Variables/Variable.tsx

```diff
@@ -49,13 +50,13 @@ function ListVariable({ name }: TemplateVariableProps) {
     }
     setVariableLoading(name, true);
     try {
-      const { data } = await variablePlugin.getVariableOptions(definition);
+      const { data } = await variablePlugin.getVariableOptions(definition, { datasources });
```

Comment:
Not important right now, but want to pick your brain about this style of dependency injection a bit more at some point. (This is just a reminder for us to talk about it at some point 😂 )